yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/ClientProperties.java

```diff
@@ -79,11 +85,11 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
 
-        YotiClientProperties that = (YotiClientProperties) o;
-        if (scenarioId != null ? !scenarioId.equals(that.scenarioId) : that.scenarioId != null) return false;
-        if (clientSdkId != null ? !clientSdkId.equals(that.clientSdkId) : that.clientSdkId != null) return false;
+        ClientProperties that = (ClientProperties) o;
+        if (!Objects.equals(scenarioId, that.scenarioId)) return false;
+        if (!Objects.equals(clientSdkId, that.clientSdkId)) return false;
 
-        return accessSecurityKey != null ? accessSecurityKey.equals(that.accessSecurityKey) : that.accessSecurityKey == null;
+        return Objects.equals(accessSecurityKey, that.accessSecurityKey);
```

Comment:
This could be 
```
return Objects.equals(scenarioId, that.scenarioId) 
         && Objects.equals(clientSdkId, that.clientSdkId)
         && Objects.equals(accessSecurityKey, that.accessSecurityKey);
```

Reply:
dropped it entirely